tests/lit-tests/discard_value_names.ispc

```diff
@@ -1,6 +1,7 @@
 
 // RUN: %{ispc} %s --target=host --nowrap -O2 --emit-llvm-text --discard-value-names -o - | FileCheck %s -check-prefix=CHECK_DISCARD
 // RUN: %{ispc} %s --target=host --nowrap -O2 --emit-llvm-text --no-discard-value-names -o - | FileCheck %s -check-prefix=CHECK_NODISCARD
+// RUN: %{ispc} %s --target=host --nowrap -O2 --emit-llvm-text -o - | FileCheck %s -check-prefix=CHECK_NODISCARD
```

Comment:
I think we need to add a test checking that by default value names are really discarded. Something like that probably should work:
```
// RUN: %{ispc} %s --target=host --nowrap -O2 --emit-asm -o - | FileCheck %s -check-prefix=CHECK_DISCARD
```

Reply:
There's no way to check that. These are purely LLVM IR values, they are not propagated anywhere further.

Reply:
Are you sure?

Reply:
I see no ways to check, unless to dump LLVM IR. My understanding is that these values are purely to make LLVM IR more human readable and are discard downstream.

Reply:
I just checked a random assembler file from my build directory.

```asm
test_uint_f___UM_vyu:                   # @test_uint_f___UM_vyu
# %bb.0:                                # %allocas
```

Is not this enough?

Reply:
Hmmm, I checked assembler and haven't noticed that. Now I see it. I'll modify the test. That's a good point.

Reply:
I've updated the files for review

Reply:
Looks good!